### PR TITLE
Creating a rough outline of the lesson plan

### PR DESCRIPTION
**Pull request by kvenkman**
I've added a few markdown files that outline the planned workflow for the rest of the "flooding and water extent" lesson of this module. I will likely fold these markdown files into the jupyter notebooks themselves to make everything cohesive, but have written them up as markdown files for ease of use.

### CODE REVIEW

File: book/2_Selecting_an_AOI.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Selecting an Area of Interest (AOI) for data analysis
+An `AOI` refers to the geographical extent of our study area, expressed in a specific co-ordinate reference frame [(CRS)](https://docs.qgis.org/3.28/en/docs/gentle_gis_introduction/coordinate_reference_systems.html#coordinate-reference-system-crs-in-detail).
```
Comment (NPalopoli) [inline book/2_Selecting_an_AOI.md:2]: Should it be "coordinate reference system" instead of "co-ordinate reference frame"?

Comment (kvenkman) [reply book/2_Selecting_an_AOI.md:2]: Yep, my bad!

Comment (PatriLoto) [reply book/2_Selecting_an_AOI.md:2]: Hi Karthik, what do you think to add some activities in the sections 3, 4 and 5. For example:

1. Getting Started
1. Selecting an AOI
1. Receiving data
**Activity A:** Choose a different area of interest, changing period and location, and then obtain data.
1. Analyzing datasets
**Activity B:** analyze the data from the new area of interest.
1. Manipulating and Visualizing datasets
**Activity C:** plot the flood map of the AOI.

Comment (kvenkman) [reply book/2_Selecting_an_AOI.md:2]: Hi Patri, I think that idea sounds good - in particular, "Activity A" makes sense as a method for learners to get used to the idea of changing the query to suit their needs.

Activity B and C are the default contents of the corresponding sections.

File: book/4_Analyzing_Datasets.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Time-series analysis of datasets
+Now that we are able to identify and retrieve datasets of interest, we can perform a simple time serieis analysis of our dataset - for example, we will try to answer the question, "How does the amount of water present in our scene change over time?"
```
Comment (PatriLoto) [inline book/4_Analyzing_Datasets.md:2]:
```suggestion
Now that we are able to identify and retrieve datasets of interest, we can perform a simple time series analysis of our dataset - for example, we will try to answer the question, "How does the amount of water present in our scene change over time?"
```

File: book/2_Selecting_an_AOI.md (hunk 1 above)
Comment (PatriLoto) [reply book/2_Selecting_an_AOI.md:2]: I apologise; I made the comment in the wrong section. So I will update it.

**Review by PatriLoto — APPROVED** [review]
The structure of the tutorial seems to me to be adequate and has the right amount of sections.

**Pull request merged**